deeplake/core/transform/transform.py

```diff
@@ -241,6 +243,15 @@ def my_fn(sample_in: Any, samples_out, my_arg0, my_arg1=0):
                 load_meta(target_ds)
                 target_ds.storage.autoflush = initial_autoflush
 
+            if not kwargs.get("disable_rechunk"):
+                rechunk_tensors = check_rechunk(target_ds.root)
+                if rechunk_tensors:
+                    if progressbar:
+                        logger.info(f"Rechunking tensors: {rechunk_tensors}")
```

Comment:
let's call this finalizing Deep Lake compute or something, rechunking might be confusing for users